files/scripts/arp_update

```diff
@@ -188,22 +188,26 @@ while /bin/true; do
   # resolve neighbor entries from CONFIG_DB in case of mismatch with kernel
   DBNEIGH="$DBNEIGH $(sonic-db-cli CONFIG_DB keys NEIGH* | sed -e 's/|/:/g')"
 
-  KERNEIGH4=$(ip -4 neigh show | grep Vlan | cut -d ' ' -f 1,3  --output-delimiter=',')
-  KERNEIGH6=$(ip -6 neigh show | grep -v fe80 | grep Vlan | cut -d ' ' -f 1,3  --output-delimiter=',')
-  for neigh in $DBNEIGH; do
-      intf="$( cut -d ':' -f 2 <<< "$neigh" )"
-      ip="$( cut -d ':' -f 3- <<< "$neigh" )"
-      if [[ $intf == *"Vlan"* ]]; then
-          if [[ $ip == *"."* ]] && [[ ! $KERNEIGH4 =~ "${ip},${intf}" ]]; then
-              pingcmd="timeout 0.2 ping -I $intf -n -q -i 0 -c 1 -W 1 $ip >/dev/null"
-              eval $pingcmd
-              logger "mismatch arp entry, pinging ${ip} on ${intf}"
-          elif [[ $ip == *":"* ]] && [[ ! $KERNEIGH6 =~ "${ip},${intf}" ]]; then
-              ping6cmd="timeout 0.2 ping6 -I $intf -n -q -i 0 -c 1 -W 1 $ip >/dev/null"
-              eval $ping6cmd
-              logger "mismatch v6 nbr entry, pinging ${ip} on ${intf}"
-          fi
-      fi
-  done
-
+  # Only on non-DualToR subtype
+  if [[ "$SUBTYPE" != "dualtor" ]]; then
+    KERNEIGH4=$(ip -4 neigh show | grep Vlan | grep -v 'FAILED\|INCOMPLETE' | cut -d ' ' -f 1,3  --output-delimiter=',')
```

Comment:
we still need to do this check on dual TOR devices, but we should be including failed/incomplete neighbors on dual tor devices.